dlblob.js

```diff
@@ -52,6 +52,43 @@ class AlAzureDlBlob {
         return this._blobService;
     };
     
+    _findMaxDlBlobSize(dlblobs) {
```

Comment:
Perhaps something like
Math.max(...dlblobs.map(val => val.contentLength);
?

Reply:
I'd rather leave it as is because it iterates through the list only ones.